data/css/mesh.scss

```diff
@@ -49,6 +49,8 @@ $display-font-composite: 'Roboto' !default;
 
 @import 'modules/window/simple';
 
+@import 'widgets/tabButton';
```

Comment:
What about leaving the previous hamburgerBasement partial and only do the import there (like we do in the scrolling partial)? I think is easier to understand why we are importing the module rather than the widget directly. I think is also useful in the way it helps to keep track of where is the button being used.